.changeset/violet-donuts-press.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-backend': patch
+---
+
+Change the jwks url value for the oidc configuration endpoint
```

Comment:
Could prolly make this JWKS instead of add it to vocab tbh :p

Reply:
SGTM. I'll change that